recipe/meta.yaml

```diff
@@ -18,7 +18,7 @@ build:
     - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
   number: 0
   missing_dso_whitelist:  # [s390x]
-    - $RPATH/ld64.so.1    # [s390x]
+    - $RPATH/ld64.so.1    # [s390x] 
```

Comment:
```suggestion
    - $RPATH/ld64.so.1    # [s390x]
```

Reply:
That darned whitespace!! Thanks I'll make this adjustment